src/layers/heatmap-layer/heatmap-layer.js

```diff
@@ -46,21 +46,24 @@ export const heatmapVisConfigs = {
  * ]
  */
 const heatmapDensity = (colorRange) => {
-
   const scaleFunction = SCALE_FUNC.quantize;
 
+  const colors = ['#000000', ...colorRange.colors]
+
   const scale = scaleFunction()
     .domain([0, 1])
-    .range(colorRange.colors);
+    .range(colors);
 
-  return scale.range().reduce((bands, level) => {
+  const colorDensity = scale.range().reduce((bands, level) => {
     const invert = scale.invertExtent(level);
     return [
       ...bands,
       invert[0], // first value in the range
       `rgb(${hexToRgb(level).join(',')})` // color
     ]
   }, []);
+  colorDensity[1] = "rgba(0,0,0,0)"
+  return colorDensity
```

Comment:
comma

Reply:
I have added the ;
and used a single quote